Documentation/In-depth/TheInstallTool/Index.rst

```diff
@@ -94,6 +94,10 @@ description of the settings carefully at least once, so you get an
 impression of what you can configure. Normally you *can*, but you *don't
 have to* change anything here during installation, as the previous steps
 took care of the most important settings.
+Pay attention to use HTTPS for your website and set :code:`$GLOBALS['TYPO3_CONF_VARS']['BE']['loginSecurityLevel']` to `normal`.
+However, on all websites without HTTPS you should set this to `rsa` and 
+activate the system extension `rsaauth`. Otherwise you will not be able to log
```

Comment:
Should we also mention that using that this encryption approach is deprecated and is not part of the TYPO3 core anymore? https://github.com/FriendsOfTYPO3/rsaauth#current-state

Reply:
Yes, good idea.

Reply:
I wouldn't add that in the 8.7 docs- that's part of the upgrade process I think.

Reply:
Indeed, I forgot the v8 context. This indeed only applies to v9+.